consensus/istanbul/backend/announce.go

```diff
@@ -419,3 +516,125 @@ func (sb *Backend) regossipIstAnnounce(msg *istanbul.Message, payload []byte, an
 
 	return nil
 }
+
+// ===============================================================
+//
+// define the AnnounceVersion messge format, the getAnnounceVersions send and handle function, and the announceVersions handle function
+
+type announceVersion struct {
+	ValAddress           common.Address
+	AnnounceMsgTimestamp uint
+}
+
+// EncodeRLP serializes av into the Ethereum RLP format.
+func (av *announceVersion) EncodeRLP(w io.Writer) error {
+	return rlp.Encode(w, []interface{}{av.ValAddress, av.AnnounceMsgTimestamp})
+}
+
+// DecodeRLP implements rlp.Decoder, and load the ee fields from a RLP stream.
```

Comment:
Should `ee` be `announceVersion`?
```suggestion
// DecodeRLP implements rlp.Decoder, and load the announceVersion fields from a RLP stream.
```